DataGridSample/DataGridSample/Views/MainPage.xaml.cs

```diff
@@ -14,5 +15,10 @@ public MainPage()
 		{
 			InitializeComponent();
 		}
+
+		private void DataGrid_ItemDoubleTapped(object sender, ItemTappedEventArgs e)
+		{
+		    (this.BindingContext as MainViewModel).GridDoubleTapped(e);
```

Comment:
Calling ViewModel fromXaml is not a good approach. If there is a requirement please create an ICommand.